node/silkworm/downloader/packets/BlockBodiesPacket.hpp

```diff
@@ -30,10 +30,6 @@ struct BlockBodiesPacket66 {  // eth/66 version
 
 namespace rlp {
 
-    size_t length(const BlockBodiesPacket& from) noexcept;
-
-    void encode(Bytes& to, const BlockBodiesPacket& from);
-
```

Comment:
It is great that we now can delete these functions. 
Can we also get rid of the decode one? 
I suppose you have not deleted it because we have problems defining 
`rlp::decode(ByteView& from, vector<T>& to) `
due to the fact that the decode overload set is really a set of a template function specializations. Why it is so? I looked at it in the past but I did not found a reason for this decision.

Reply:
Right, I've created Issue #493.